carbonmark-api/src/plugins/bearer.ts

```diff
@@ -1,16 +1,27 @@
 import fastifyJwt from "@fastify/jwt";
 import fp from "fastify-plugin";
+import { isNil } from "lodash";
+
+const SECRET = process.env.JWT_SECRET;
```

Comment:
pls add to .env.example and make a note when this variable is required (only for developing on /users POST and PATCH yes?

Reply:
This is required whenever  the `authenticate` decorator is used. So not just for development but in production too. It is coming from Vercel so it shouldnt need any additional manual steps